### PR TITLE
building fundraiser page relates #10

### LINKED ISSUE #10: Building Fundraiser Section

### PR DESCRIPTION
**Pull request by ahmedkhilla**

### CODE REVIEW

**Review by ayaenshasy2002 — DISMISSED** [review]

**Review by yahya-1 — APPROVED** [review]

**Review by yahya-1 — DISMISSED** [review]

**Review by Ahmed-jebara — APPROVED** [review]

**Review by Ahmed-jebara — APPROVED** [review]

**Review by Ahmed-jebara — DISMISSED** [review]

File: fundraiser1/fundraiser1.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    <main>
+      <!-- Fundraise Section  -->
+      <section class="fundraise-section">
+        <div class="container">
+          <!-- Fundraise Picture Section  -->
+          <div class="fundraise-picture">
+            <img
+              src="/DonationAPP-team-9/asstes/photo-children.png"
+              class="children-picture"
+              alt=""
```
Comment (emanjaber) [inline fundraiser1/fundraiser1.html]: You have to edit the `src` of the image.

This is the correct way to write it:
`src="../asstes/photo-children.png"`

And the same for all the images, I will mark them with **(1)**

File: fundraiser1/fundraiser1.html [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+            />
+            <div class="bar-flex pic-bar">
+              <div class="pic-description">
+                <img src="/DonationAPP-team-9/asstes/timer-icon.png" alt="" />
```
Comment (emanjaber) [inline fundraiser1/fundraiser1.html]: **(1)**
`src="../asstes/timer-icon.png"`

File: fundraiser1/fundraiser1.html [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+              <img
+                src="/DonationAPP-team-9/asstes/photo-person5.png"
+                class="personal-pic"
+                alt="person-pic 5"
+              />
+              <img
+                src="/DonationAPP-team-9/asstes/photo-person1.png"
+                class="personal-pic"
+                alt="person-pic 1"
+              />
```
Comment (emanjaber) [inline fundraiser1/fundraiser1.html]: **(1)**
edit `src` for these 4 images

Also, you need to make the images "in-line" at mobile size

File: fundraiser1/fundraiser1.html [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+      <section class="organizer-section">
+        <div class="container">
+          <article>
+            <h4>Oraganiser</h4>
+            <div class="profile-box">
+              <img
+                src="/DonationAPP-team-9/asstes/profile-llipse.png"
+                alt="profile-pic"
+                class="profile-pic"
+              />
```
Comment (emanjaber) [inline fundraiser1/fundraiser1.html]: **(1)**
edit `src` of this image

File: fundraiser1/fundraiser1.css [hunk 5]
```diff
@@ -0,0 +1,259 @@
+@import url("https://cdn.jsdelivr.net/npm/@xz/fonts@1/serve/metropolis.min.css%22%22");
```
Comment (emanjaber) [inline fundraiser1/fundraiser1.css]: Delete `%22%22` from the end of the link, so the site will recognize the font

File: fundraiser1/fundraiser1.css [hunk 6]
```diff
@@ -0,0 +1,259 @@
...
+html {
+  font-size: 10px;
+}
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+body {
+  background-color: #e5e5e5;
```
Comment (emanjaber) [inline fundraiser1/fundraiser1.css]: Change `background-color` into `#FBFBFB` so it will be like the design

File: fundraiser1/fundraiser1.css [hunk 7]
```diff
@@ -0,0 +1,259 @@
...
+.education-btn {
+  width: 70px;
+  height: 25px;
+  background-color: #3eb489;
+  border-radius: 5px;
+  text-align: center;
+}
+.education-btn p {
+  line-height: 23px;
+}
```
Comment (emanjaber) [inline fundraiser1/fundraiser1.css]: Make the color font white
`  color: #fff;`

**Review by emanjaber — CHANGES_REQUESTED** [review]
Great job Ahmad!
Make the changes I mentioned

**Review by ayaenshasy2002 — APPROVED** [review]

**Review by emanjaber — APPROVED** [review]
**Well done Ahmad! 🎉**

**Review by Ahmed-jebara — APPROVED** [review]

**Review by yahya-1 — APPROVED** [review]

**Pull request merged**